index.html

```diff
@@ -0,0 +1,101 @@
+                </svg> 
+            </div>
+      
+          <div class="col-md-4 col-xs-12 pt-2 copyright">
+            <p> © 2021 NEWSWEEK DIGITAL LLC</p>  
+          </div>
+      
+          <div class="col-md-5 col-xs-12">
+            <ul class="footer-1-socials d-flex align-items-center justify-content-md-end">
+              <li> <i class="fa fa-facebook-square"></i> </li>
+              <li> <i class=" mx-2 fa fa-twitter"></i> </li>
+              <li> <i class=" mx-2 fa fa-instagram"></i> </li>
+              <li> <i class=" mx-2 fa fa-twitter"></i> </li>
+              <li> <i class=" mx-2 fa fa-pinterest"></i> </li>
+            </ul>
```

Comment:
Social media icons should have a link.

Reply:
We have effected all changes @teekaytech, thank you for your review